src/StaticFAQ/ArticleFeedback/FAQArticleFeedback.js

```diff
@@ -3,19 +3,21 @@
 import * as React from 'react';
 import css from 'styled-jsx/css';
 
-import ScreenInitial from './ScreenInitial';
-import ScreenInput from './ScreenInput';
+import ScreenVoting from './ScreenVoting';
+import ScreenFeedback from './ScreenFeedback';
+import ScreenAdditionalFeedback from './ScreenAdditionalFeedback';
 import ScreenThankyou from './ScreenThankyou';
 import ScreenError from './ScreenError';
-import screensList from './screensList';
+import screenList from './screenList';
```

Comment:
Why `screenList` and not `ScreenList`?  (You have `ScreenVoting`, `ScreenFeedback` etc.)

Reply:
because it's just an exported object and not a component, in my opinion it would be quite misleading in PascalCase